src/test/evo_trivialvalidation.cpp

```diff
@@ -29,7 +29,7 @@ void TestTxHelper(const CMutableTransaction& tx, bool is_basic_bls, bool expecte
     if (payload_to_fail) return;
 
     TxValidationState dummy_state;
-    BOOST_CHECK_EQUAL(payload.IsTriviallyValid(is_basic_bls, dummy_state), !expected_failure);
+    BOOST_CHECK_EQUAL(payload.IsTriviallyValid(is_basic_bls, is_multi_payout_active, dummy_state), !expected_failure);
```

Comment:
I think  the  list of `trivially_valid` and `trivially_invalid` should have at least one transaction of each type with multiple payees to be sure that serialization works correctly.

Reply:
moved most of the unit test coverage to `trivially_valid` `trivially_invalid`